docker/.dockerignore

```diff
@@ -0,0 +1,5 @@
+.env
+.dockerignore
```

Comment:
It is useless to explicitly ignore `.env` and `.dockerignore`: there are ignored by default.

By the way, you don't need to create such a file because in your Dockerfile, you copy explicitly the file you want to add to the Docker image.